stack/indexer/deb/debian/postinst

```diff
@@ -14,12 +14,13 @@ export USER=${NAME}
 export GROUP=${NAME}
 export CONFIG_DIR=/etc/${NAME}
 export INSTALLATION_DIR=/usr/share/${NAME}
+export BACKUP_DIR=/usr/share/backup
```

Comment:
I think `/usr/share/backup` is too general, maybe `/usr/share/wazuh-backup` is better